install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -360,10 +360,11 @@ data:
 {{- else }}
   operator-api-serve-addr: '[::1]:9234'
 {{- end }}
-{{ if gt (len .Values.global.hubble.listenAddresses) 0 }}
-  # A space separated list of addresses for Hubble server to listen to. Currently only unix
-  # domain sockets are supported. Hubble gRPC endpoint will be disabled if this field is not
-  # set.
+{{ if .Values.global.hubble.enabled }}
```

Comment:
where is this block being closed?

Reply:
line 391.

Reply:
I've opened #10826 and #10827 because of this

Reply:
fixed